README.md

```diff
@@ -1 +1,140 @@
-# Leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [To Do list] <a name="list structure"></a>
+
+>**ToDo List!** is my fourth project at Microverse, in this project I have created a basic list of tasks where the user can add a new task, mark as completed and delete all the completed tasks 
```

Comment:
- [ ] Your content for this section is not reflecting what your current project should be on, you are now working on the `leaderboard` project not the `todolist` please kindly make the necessary corrections so your readme is professional. :+1: :sparkles:

Reply:
Done